openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -178,4 +179,4 @@ def patch(self, request, username):
                 status=status.HTTP_400_BAD_REQUEST
             )
 
-        return Response(status=status.HTTP_204_NO_CONTENT)
+        return Response(account_settings)
```

Comment:
@nasthagiri Am I looking at the correct thing? Existing doc seems to already show the return value as 200 for PATCH: http://edx.readthedocs.org/projects/edx-platform-api/en/latest/user/accounts.html#get-and-update-the-user-s-account-information